archspec/cpu/detect.py

```diff
@@ -77,20 +77,49 @@ def proc_cpuinfo():
     return info
 
 
+@info_dict(operating_system="Windows")
+def proc_py_cpuinfo():
+    """Returns a raw info dictionary by using py-cpuinfo"""
+    try:
+        import cpuinfo
+    except ImportError:
+        return {}
```

Comment:
Linter is complaining that this import is not at the top-level. Should I explicitly ignore that warning?